cloudflare/resource_cloudflare_custom_hostname_fallback_origin_test.go

```diff
@@ -0,0 +1,46 @@
+package cloudflare
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccCloudflareCustomHostnameFallbackOrigin(t *testing.T) {
```

Comment:
We're only testing the `Create` method here. Should we also be adding coverage for `Update`, `Delete` and `Import`?

Reply:
I've added tests for Update. There hasn't been any other Delete or Import tests that I could see in the rest of the codebase. I'll have a punt at writing a delete one but not sure where to start for Import.

Reply:
I added checkdestroy to all of the tests.